.vscode/tasks.json

```diff
@@ -1,49 +1,28 @@
-// See https://go.microsoft.com/fwlink/?LinkId=733558
-// for the documentation about the tasks.json format
 {
     "version": "2.0.0",
     "tasks": [
```

Comment:
Nit: The build pipeline for this extension looks for a task to run "tslint" (which should now be eslint) which makes sure that all the code passes eslint.

Can you add back a task to run "eslint"?

I've disabled the task in our build pipeline for now, but I'd like to re-enable it.

Reply:
Can the pipeline directly do "npm run lint" (which exists)? Or must it run via a VS Code task?

---
In reply to: [441522219](https://github.com/microsoft/sarif-vscode-extension/pull/282#discussion_r441522219) [](ancestors = 441522219)

Reply:
Resolved with Gabe offline. `npm run list` was sufficient.

---
In reply to: [441709918](https://github.com/microsoft/sarif-vscode-extension/pull/282#discussion_r441709918) [](ancestors = 441709918,441522219)